packages/frontend-v2/components/Sidebar/SidebarSection.tsx

```diff
@@ -38,25 +42,42 @@ const SidebarSection: React.FC<SidebarSectionProps> = ({
         _active={{
           backgroundColor: "neutral.200",
         }}
+        display="flex"
+        position="sticky"
```

Comment:
Made the sidebar section headers sticky, so you can see what section you're in even when you've scrolled down :)

Reply:
NAICE